packages/terra-data-grid/src/subcomponents/Cell.module.scss

```diff
@@ -4,15 +4,15 @@
 @import '../clinical-lowlight-theme/Cell.module';
```

Comment:
Should now be able to remove disabling the linter rule in the scss files in this directory